pkg/fileservice/local_fs.go

```diff
@@ -111,6 +111,10 @@ func NewLocalFS(
 	return fs, nil
 }
 
+func (l *LocalFS) SetRootpath(path string) {
```

Comment:
I don't understand why this is required.
After initialization, a LocalFS should never change its root path.